common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -55,6 +58,7 @@ public final class CharacterModel extends Model {
     private boolean inCharacterSelection;
     private boolean hasCharacter;
 
+    private Boolean silverbullSubscriber = null;
```

Comment:
perhaps you should move this to after level. I think that is a more ergonomic approach to these variables. Don't forget to fix the toString.

Reply:
Also, if you do the change I suggested in emeraldModel, I see no reason why this has to be a boxed type.

Reply:
Boxed type ensures we only check it once per game launch, but maybe I can improve this with magicus' idea of checking the remaining subscription duration.